js/menus/menuTemplate.js

```diff
@@ -184,17 +186,35 @@ function getTemplate(app) {
             checked: launchOnStartup,
             click: function (item) {
                 if (item.checked){
-                    symphonyAutoLauncher.enable()
-                    .catch(function (err) {
-                        let title = 'Error setting AutoLaunch configuration';
-                        electron.dialog.showErrorBox(title, title + ': ' + err);
-                    });
+                    if (isMac){
+                        childProcess.exec(`launchctl load ${launchAgentPath}`, (err) => {
```

Comment:
why do we do different work to change how to enable/disable startup for mac?  doesn't the auto-launch package work for mac?  i thought it should take care of all of this???